pynest/nest/lib/hl_api_types.py

```diff
@@ -667,11 +674,18 @@ def format_row_(s, t, sm, w, dly):
         d_h = 'delay'
 
         # Find maximum number of characters for each column, used to determine width of column
-        src_len = max(len(src_h) + 2, floor(log(max(srcs), 10)))
-        trg_len = max(len(trg_h) + 2, floor(log(max(trgt), 10)))
-        sm_len = max(len(sm_h) + 2, len(max(s_model, key=len)))
-        w_len = len(w_h) + 2
-        d_len = len(d_h) + 2
+        if len(params) == 0:
+            src_len = len(src_h) + 2
+            trg_len = len(trg_h) + 2
+            sm_len = len(sm_h) + 2
+            w_len = len(w_h) + 2
+            d_len = len(d_h) + 2
+        else:
+            src_len = max(len(src_h) + 2, floor(log(max(srcs), 10)))
+            trg_len = max(len(trg_h) + 2, floor(log(max(trgt), 10)))
+            sm_len = max(len(sm_h) + 2, len(max(s_model, key=len)))
+            w_len = len(w_h) + 2
+            d_len = len(d_h) + 2
```

Comment:
Maybe something like this instead, to shorten somewhat?
```suggestion
        else:
            src_len = max(len(src_h) + 2, floor(log(max(srcs), 10)))
            trg_len = max(len(trg_h) + 2, floor(log(max(trgt), 10)))
            sm_len = max(len(sm_h) + 2, len(max(s_model, key=len)))
        w_len = len(w_h) + 2
        d_len = len(d_h) + 2
```